[
  {
    "cmd": [
      "vpython",
      "-u",
      "RECIPE_MODULE[recipe_engine::file]/resources/fileutil.py",
      "--json-output",
      "/path/to/tmp/json",
      "ensure-directory",
      "--mode",
      "0777",
      "[CACHE]/builder/w"
    ],
    "infra_step": true,
    "name": "ensure work_dir"
  },
  {
    "cmd": [],
    "name": "https://chromium.googlesource.com/fail",
    "~followup_annotations": [
      "@@@STEP_FAILURE@@@"
    ]
  },
  {
    "cmd": [
      "git_cache.py",
      "populate",
      "--reset-fetch-config",
      "--ref",
      "refs/branch-heads/*",
      "--break-locks",
      "--cache-dir",
      "[CACHE]/builder/w",
      "--verbose",
      "https://chromium.googlesource.com/fail"
    ],
    "cost": {
      "cpu": 500,
      "disk": 20,
      "memory": 50,
      "net": 0
    },
    "env": {
      "CHROME_HEADLESS": "1",
      "GIT_HTTP_LOW_SPEED_LIMIT": "0",
      "GIT_HTTP_LOW_SPEED_TIME": "0",
      "OVERRIDE_BOOTSTRAP_BUCKET": "experimental-gs-bucket"
    },
    "env_suffixes": {
      "PATH": [
        "RECIPE_REPO[depot_tools]"
      ]
    },
    "name": "https://chromium.googlesource.com/fail.populate",
    "~followup_annotations": [
      "@@@STEP_NEST_LEVEL@1@@@",
      "@@@STEP_FAILURE@@@"
    ]
  },
  {
    "failure": {
      "failure": {},
      "humanReason": "Updated cache for 0/1 repos.\n\nEncountered failures for 1 repos:\n\n  * https://chromium.googlesource.com/fail"
    },
    "name": "$result"
  }
]